book/src/concepts/use_cases.md

```diff
@@ -0,0 +1,53 @@
+# Use Cases
+
+The following are examples of people who are well-suited to using a Portal Network client, like Trin.
+
+## Laptop wallet user
+
+A user has a laptop that frequently is turned off. When
+they want to transact, they can turn on Trin and connect their
+wallet to it.
+
+*Benefit*: Wallet use without reliance on third party wallet APIs.
```

Comment:
```suggestion
Traditionally it hasn't be reasonable to run on fullnode and access Ethereum Protocol on laptops due to 
- storage
- bandwidth
- battery life
- long sync times
requirements. If a user was to restart there computer and wanted to send a transaction they would need to wait hours. Portal solves this by using minimal storage, battery life, bandwidth, and Portal also has instant sync times.

*Benefit*: Wallet use without reliance on third party wallet APIs.
```

I don't think my suggestion is perfect by any means, but I think this `use case` is missing why running fullnodes doesn't work and how Portal is solving a problem. These examples assume the user knows a lot of information about Ethereum and it's problems, I think Trin is targeting a much wider audience so assuming to knowledge isn't reasnable

Reply:
Right, I didn't actually update this particular section at all. I was more focused on the intro/quickstart stuff. Sorry for the how the page copy made that unclear.

I do think it's nice to have a page dedicated to punchy user stories that doesn't re-hash the overall benefits. But also it looks like we're missing a page that dives into the overall benefits. I'll throw a new page in, but I'll consider it out of scope for this PR to get it super polished.